src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -38,6 +39,29 @@ import {
   makeSortedAccountBalances,
 } from './utils'
 
+type ParamFilter = {
```

Comment:
There is duplication in here and `stakingDataSlice` on these types and selector creators for now, but it doesn't matter, `stakingDataSlice` gets entirely removed in the next PR.

Reply:
I came here to say we should extract them into a common type, but then found this helpful comment 😀